assets/scripts/datalayer/common.js

```diff
@@ -44,10 +44,12 @@ document.shopAnalytics = {
       if (product_data.list) {
         product.list = product_data.list;
       }
-      products_data.push(product);
+      // It can happen that the hidden markup is cloned in the DOM (like in a
+      // slider), we only want it tracked once.
+      products_data[product_data.list + ':' + product_data.position + ':' + product.id] = product;
```

Comment:
Do we still need this?  I'd rather not introduce this change right now, because the PM/QA will probably not test it.

Reply:
Ok, it is out.